index.html

```diff
@@ -145,6 +145,11 @@ <h2>
         For example, the badge content '7' should be displayed as '7' in the
         <a data-cite="ltli/#locale">locale</a> 'en-NZ' but as '٧' in 'ar-EG'.
       </p>
+      <p class="note">
+        The user agent MAY clear the badge at any time, though this
```

Comment:
```suggestion
        The user agent can clear the badge at any time, though this
```

Avoid RFC2119 keywords in non-normative sections.